### PR TITLE
Planetstation: Build over chasms

### PR DESCRIPTION
**Pull request by Iamgoofball**
see title

rods + tile as usual

requested by kor

picture for oranges:

[brazzers-went-to-4chan-users-and-asked-them-to-write-a-script-the-result-is-meme-lover-and-its-100-wtf_1](https://cloud.githubusercontent.com/assets/4081722/23977026/d08e08bc-09a8-11e7-830c-de33c63c066d.jpg)

### CODE REVIEW

**Comment by KorPhaeron** [comment]
Thanks

File: code/game/turfs/simulated/chasm.dm [hunk 1]
```diff
@@ -21,6 +21,34 @@
...
+			var/obj/item/stack/tile/plasteel/S = C
+			if(S.use(1))
+				qdel(L)
+				playsound(src, 'sound/weapons/Genhit.ogg', 50, 1)
+				to_chat(user, "<span class='notice'>You build a floor.</span>")
+				ChangeTurf(/turf/open/floor/plating)
+			else
+				to_chat(user, "<span class='warning'>You need one floor tile to build a floor!</span>")
+		else
+			to_chat(user, "<span class='warning'>The plating is going to need some support! Place metal rods first.</span>")
```
Comment (Cyberboss) [inline code/game/turfs/simulated/chasm.dm]: I'd prefer if you refactored the lattice building into a proc on turf/open instead of the copypasta

Comment (Iamgoofball) [inline code/game/turfs/simulated/chasm.dm]: Nope

**Comment by Armhulen** [comment]
new label hype!

**Comment by PKPenguin321** [comment]
>cant spawn chasms on station to permanently grief players anymore

why live?

**Comment by Iamgoofball** [comment]
>planetstation

**Comment by calzilla1** [comment]
Make super chasms that can't be built over

**Comment by Iamgoofball** [comment]
24805750-outdoor-daytime-viewing-telescope-in-rural-area-front-view-blue-sky-and-clouds-reflection-in-scope-g-stock-photo

File: code/game/turfs/simulated/chasm.dm (hunk 1 above)
Comment (Iamgoofball) [inline code/game/turfs/simulated/chasm.dm]: cyberboss said in coderbus that he'll update this on his own

**Comment by Repukan** [comment]
Is that screen cap from that meme porn

**Comment by ExcessiveUseOfCobblestone** [comment]
This is a nerf to that one clown shuttle that has greentext in it.

**Comment by Cheridan** [comment]
the code here is bad but cyberboss says he'll fix in it ##24340 so i'll just merge

File: code/game/turfs/simulated/chasm.dm (hunk 1 above)
Comment (optimumtact) [inline code/game/turfs/simulated/chasm.dm]: the fuck, this isn't an excuse to let shit code into the repo @Cheridan

Comment (Iamgoofball) [inline code/game/turfs/simulated/chasm.dm]: cyberboss said he'd do it

Comment (optimumtact) [inline code/game/turfs/simulated/chasm.dm]: his pr is massive, there's little to no guarantee he will ever finish it.

Comment (Iamgoofball) [inline code/game/turfs/simulated/chasm.dm]: fuck

**Pull request merged**